Assets/_Functionalities/AssetBundles/Scripts/AssetBundleLoader.cs

```diff
@@ -71,17 +71,22 @@ private void FixShadersForEditor(GameObject asset)
             MeshRenderer[] renderers = asset.GetComponentsInChildren<MeshRenderer>();
             foreach (MeshRenderer renderer in renderers)
             {
-                if (renderer.material != null)
+                if (!IsPrefab(asset) && renderer.material)
                 {
                     renderer.material.shader = Shader.Find(renderer.material.shader.name);
+                    continue;
                 }
-                else
-                {
-                    renderer.sharedMaterial.shader = Shader.Find(renderer.sharedMaterial.shader.name);
-                }
+
+                renderer.sharedMaterial.shader = Shader.Find(renderer.sharedMaterial.shader.name);
             }
         }
 
+        private static bool IsPrefab(GameObject gameObject)
+        {
+            // A prefab will have a `transform` but no parent if it's a root prefab in play mode
+            return gameObject.scene.name == null;
```

Comment:
name is een string, en die is nooit null, dus moet dit `return gameObject.scene.name == string.Empty` zijn?